### PR TITLE
Feat/example semantic article headline

### PR DESCRIPTION
**Pull request by colincclark**
Proposal for Issue https://github.com/newsuk/times-components/issues/606

Added `accessibility` attributes for better semantics for the article-summary headline. If this canbe approved, will roll it out across related articles / more packages as appropriate

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -82,8 +82,5 @@
     -moz-osx-font-smoothing: grayscale;
   }
 
-  h1, h2, h3, h4 {
-      -webkit-font-smoothing: auto;
```
Comment (colincclark) [inline .storybook/preview-head.html]: we should not be styling `<hX>` tags in storybook

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c9984c85edf97fbd79d54a2c0b74150d86658367/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b015f64028258375c03ff6b5f6a372511d815118/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/9481889bbe19cb74597a744e9073184b9f360fb9/dextrose-test/index.html

File: .storybook/preview-head.html (hunk 1 above)
Comment (craigbilner) [reply .storybook/preview-head.html]: @colincclark this is so storybook looks how it will on the site/render

**Comment by craigbilner** [comment]
@colincclark did we not just need to use the [correct accessibility props](https://github.com/necolas/react-native-web/blob/master/website/guides/accessibility.md#accessibilityrole)?

File: .storybook/preview-head.html (hunk 1 above)
Comment (colincclark) [reply .storybook/preview-head.html]: @craigbilner 👍 for the `accessible` knowledge - would love to role this out to article and related article on Monday. However, leaving this style here in storybook makes the font look horrible for these H elements. Any other ideas how we can remove or override these styles? Removing them did not seem to adversely affect the storybook components AFAICS....

**Pull request closed without merging**